### DISCUSSION TITLE
Ladies and Gentlemen...Our Discord Server Has Launched!

### DISCUSSION THREAD

**Discussion by daley-mottley** [dmotts/yt-to-cloud-uploader#37] [Announcements]
Hey everyone! Our private Discord server is ready!

Please message me if you would like to join in! 😄🙌

**Comment by daley-mottley** [comment]
@swarupn17 @HomerusJa @shawaiz-khan @Brijeshthummar02 @shaikmynuddin @NimraAslamkhan @Alphx-rgb @krtEngineer

**Reply by HomerusJa** [reply]
Where have you been? We couldnt find this repo nor your account for the last few days! I was just going over creating a new repo. What happened?

**Reply by daley-mottley** [reply]
Hey @HomerusJa, there was a hiccup for a moment. Contacted support and now we're back online. So, without further ado...let's get this party started!!!! 😁🎊🙌

**Comment by HomerusJa** [comment]
And can you provide the link to the server somewhere?

**Reply by HomerusJa** [reply]
If you want to have the server private, I'd argue that this is an open-source project, so everyone should be allowed too join the server. Also, like that, it is easiest for new contributors to jumo in.

**Reply by daley-mottley** [reply]
This is a "soft" launch. Keeping everything relatively small for now (I know it's open source, however I believe there should be some sort of structure at the beginning stages so a clear path could be set for our future contributors to follow. Then, we'll go public as the project gets larger. How does that sound to you @HomerusJa?

**Reply by HomerusJa** [reply]
We can do that, definitely.

**Reply by HomerusJa** [reply]
How do you want to let us join? My discord username is HomerusJa too.

**Comment by shaikmynuddin** [comment]
> Hey everyone! Our private Discord server is ready!
> 
> Please message me if you would like to join in! 😄🙌
I wanna join
my username shaikmynuddin

**Reply by daley-mottley** [reply]
Invite sent @shaikmynuddin, please check your 📬

**Comment by swarupn17** [comment]
Hey @dmotts can you add me in the Discord server. My username is swarup17

**Comment by krtEngineer** [comment]
Hi. @dmotts please add me in discord server. My username is: kushagra_raj_tiwari
